### PR TITLE
fixed #1236 issue in main repo

### LINKED ISSUE #1236: GRPC Executor dosn`t close connection after exec
**Describe the bug**
GRPC Executor doesn t close connection after exec 

**To Reproduce**
1. Create grpc job executor with schedule 1s 
2. Wait some time for enough tasks completed
3. Make command `netstat -nep | grep ESTABLISHED| wc -l` in cli on container (docker), we get many open connection

**Expected behavior**
After job exec connection over grpc must be closed

**Additional context**
I have think about problem in after error check https://github.com/distribworks/dkron/blob/master/builtin/bins/dkron-executor-grpc/grpc.go#L94
This line doesn t have `defer cc.Close()`

### PR DESCRIPTION
**Pull request by edwsel**
Fixed close connection when using grpc executor

### CODE REVIEW

**Review by yvanoers — CHANGES_REQUESTED** [review]
Minor styling request, but otherwise LGTM, thanks!

File: builtin/bins/dkron-executor-grpc/grpc.go [hunk 1]
```diff
@@ -32,12 +33,13 @@ type GRPC struct{}
...
-//     "timeout": "30",                               // Request timeout, unit seconds
-//     "expectCode": "0",                             // Expect response code, any of the described here https://grpc.github.io/grpc/core/md_doc_statuscodes.html
-// }
+//
+//	"executor_config": {
+//	    "url": "127.0.0.1:9000/demo.DemoService/Demo", // Request url
+//	    "body": "",                                    // POST body
+//	    "timeout": "30",                               // Request timeout, unit seconds
+//	    "expectCode": "0",                             // Expect response code, any of the described here https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+//	}
```
[the remark this answers is not in the document]
Comment (edwsel) [reply builtin/bins/dkron-executor-grpc/grpc.go:41]: fixed

[the remark this answers is not in the document]
Comment (yvanoers) [reply builtin/bins/dkron-executor-grpc/grpc.go:41]: Thanks, though, that was only the blank line - the indenting still doesn't align. with line 35.

[the remark this answers is not in the document]
Comment (edwsel) [reply builtin/bins/dkron-executor-grpc/grpc.go]: Sorry, my ide auto format code, fixed

**Review by vcastellm — APPROVED** [review]
LGTM thx!

**Pull request merged**